src/DurableTask.Generators/AzureFunctions/SyntaxNodeUtility.cs

```diff
@@ -1,29 +1,33 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.Collections.Generic;
-using System.Linq;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 namespace Microsoft.DurableTask.Generators.AzureFunctions
 {
-    public static class SyntaxNodeUtility
+    static class SyntaxNodeUtility
     {
         public static bool TryGetFunctionName(SemanticModel model, MethodDeclarationSyntax method, out string? functionName)
         {
+            functionName = null;
```

Comment:
nit: I prefer to leave this near the return statement so that the compiler can help me know when I've forgotten to correctly assign it to a non-null value.